README.md

```diff
@@ -1 +1,98 @@
-# Budget-App
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Budget App
+
+> This is a mobile web application, that allows you to manage your budget. The budget app lists all transactions associated with a category, and this allows the user to see how much money the user has spent.
+
+## Live Link
+
+- [Heroku](https://coolest-budget-app.herokuapp.com/)
+
+## Video
+
+- [Video](https://www.loom.com/share/6ce30ae3413b4e52ba1ea1cf37ae5eb4)
+
+## Built With
+
+- Ruby on Rails
+- Postgresql
+- Ruby on Rails Gems
+- Bootstrap
+- Sass
+
+---
+## Getting Started
+
+> To get a local copy up and running please follow these steps:
+
+- Open the command prompt in this directory.
+
+- Clone of the Repo:
+
+      $ git clone https://github.com/hamid-murambiwa/Budget-App.git
+
+- Go to the repository folder in your command prompt and navigate into the project file:
+
+      $ cd Budget-App
+
+- Install the Ruby on Rails Gems:
+
+      $ bundle install or bundle i
+
+- To analyze and format the applictaion code run:
+
+      $ rubocop -A
+
+- Start the server:
+
+      $ rails s
+
+---
+
+## To run the tests
+
+- Install rspec:
+
+      $ bundle install
+
+- and
+
+      $ rails generate rpesc:install
+
+- run all rspec tests:
+
+      $ rspec spec
+
+
+---
+## Prerequisites
+
+- Ruby
+- Ruby on Rails
+- Postgresql
+
+---
+## Author
+
+👤 **Hamid Murambiwa**
+
+- GitHub: [@hamid-murambiwa](https://github.com/hamid-murambiwa/)
+- Twitter: [@Hamid87789454](https://twitter.com/Hamid87789454/)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/hamid-murambiwa/)
+
+## 🤝 Contributing
+
+>Contributions, issues, and feature requests are welcome!
+>Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+>Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- This project built in Microverse's Fourth Module's Capstone.
+
```

Comment:
- [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/)[](https://github.com/hamid-murambiwa/Budget-App/pull/5/files#live-version).